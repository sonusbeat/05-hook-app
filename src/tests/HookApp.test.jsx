import { shallow } from 'enzyme';
import HookApp from '../HookApp';

describe("Puebas en <HookApp />", () => {
  test("Debe coincidir con el snapshot", () => {
    const wrapper = shallow(<HookApp />);

    expect(wrapper).toMatchSnapshot();
  });
});
