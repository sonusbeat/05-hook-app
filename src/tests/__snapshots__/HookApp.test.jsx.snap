// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Puebas en <HookApp /> Debe coincidir con el snapshot 1`] = `
<div>
  <h1>
    Hook App
  </h1>
</div>
`;
